spec/chewy/index/import/routine_spec.rb

```diff
@@ -11,8 +11,8 @@
     CitiesIndex.create!
   end
 
-  let(:index) { [double(id: 1, name: 'Name', object: {}), double(id: 2, name: 'Name', object: {})] }
-  let(:delete) { [double(id: 3, name: 'Name')] }
+  let(:index) { [double('to_index', id: 1, name: 'Name', object: {}), double(id: 2, name: 'Name', object: {})] }
+  let(:delete) { [double('to_delete', id: 3, name: 'Name', object: {})] }
```

Comment:
Why? Anyway, if we're using a name for the first element in `index` array - let's do the same with the second too.

Reply:
It's easier to debug  unexpected messages sent to double if the doubles are named.